diff --git a/node_modules/@loopring-web/loopring-sdk/dist/index.js b/node_modules/@loopring-web/loopring-sdk/dist/index.js
index 90a3dd8..d8cab95 100644
--- a/node_modules/@loopring-web/loopring-sdk/dist/index.js
+++ b/node_modules/@loopring-web/loopring-sdk/dist/index.js
@@ -1,8 +1,4 @@
 
 'use strict'
 
-if (process.env.NODE_ENV === 'production') {
-  module.exports = require('./loopring-sdk.cjs.production.min.js')
-} else {
-  module.exports = require('./loopring-sdk.cjs.development.js')
-}
+module.exports = require('./loopring-sdk.cjs.development.js')
diff --git a/node_modules/@loopring-web/loopring-sdk/dist/loopring-sdk.cjs.development.js b/node_modules/@loopring-web/loopring-sdk/dist/loopring-sdk.cjs.development.js
index 1fb1c98..61740f6 100644
--- a/node_modules/@loopring-web/loopring-sdk/dist/loopring-sdk.cjs.development.js
+++ b/node_modules/@loopring-web/loopring-sdk/dist/loopring-sdk.cjs.development.js
@@ -4,10 +4,10 @@ Object.defineProperty(exports, '__esModule', { value: true });
 
 function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }
 
-require('core-js/modules/es.promise.js');
-require('core-js/modules/es.typed-array.set.js');
-require('core-js/modules/es.typed-array.sort.js');
-require('core-js/modules/es.array.sort.js');
+// require('core-js/modules/es.promise.js');
+// require('core-js/modules/es.typed-array.set.js');
+// require('core-js/modules/es.typed-array.sort.js');
+// require('core-js/modules/es.array.sort.js');
 var ethUtil = require('ethereumjs-util');
 var BN = _interopDefault(require('bn.js'));
 var BigNumber = _interopDefault(require('bignumber.js'));
diff --git a/node_modules/@loopring-web/loopring-sdk/dist/loopring-sdk.esm.js b/node_modules/@loopring-web/loopring-sdk/dist/loopring-sdk.esm.js
index a7065d0..c3329c0 100644
--- a/node_modules/@loopring-web/loopring-sdk/dist/loopring-sdk.esm.js
+++ b/node_modules/@loopring-web/loopring-sdk/dist/loopring-sdk.esm.js
@@ -1,13 +1,13 @@
-import 'core-js/modules/es.promise.js';
-import 'core-js/modules/es.typed-array.set.js';
-import 'core-js/modules/es.typed-array.sort.js';
-import 'core-js/modules/es.array.sort.js';
+// import 'core-js/modules/es.promise.js';
+// import 'core-js/modules/es.typed-array.set.js';
+// import 'core-js/modules/es.typed-array.sort.js';
+// import 'core-js/modules/es.array.sort.js';
 import { toBuffer as toBuffer$1, toChecksumAddress, sha256, keccak, hashPersonalMessage, fromRpcSig, ecrecover, pubToAddress, keccak256, toRpcSig } from 'ethereumjs-util';
 import BN from 'bn.js';
 import BigNumber from 'bignumber.js';
 import { Buffer as Buffer$1 } from 'buffer';
 import axios from 'axios';
-import 'core-js/modules/es.object.from-entries.js';
+// import 'core-js/modules/es.object.from-entries.js';
 import crypto from 'crypto-js';
 import { soliditySHA3, methodID, rawEncode, rawDecode } from 'ethereumjs-abi';
 import { TypedDataUtils } from 'eth-sig-util';
